tests/test_client.py

```diff
@@ -48,7 +107,17 @@ def test_batch_geocode(self):
         assert res[1]['city'] == 'Essen'
         assert res[2]['country'] == 'Allemagne'
 
-    def test_batch_reverse_geocode(self):
+    def test_batch_reverse_geocode(self, monkeypatch):
+        class MockReqestsGet:
```

Comment:
Typo: MockReqUestsGet

Also: test fails because "url" and "headers" must be an argument in the __init__

Also: after fixing __init__ arguments, you will see that the test still fails. The current result contains twice the expected result. Check with the debugger